src/app/resources/resources.component.html

```diff
@@ -1,52 +1,56 @@
-          <ng-template ngFor let-col [ngForOf]="item.doc.collection">{{ col }} / </ng-template>
-        </ng-template>
+        Author: {{ element.author }},Year: {{ element.year }}, Media: {{ element.Medium }}, Language: {{ element.language }}, Subject/s: {{element.subject}}, Publisher/Attribution: {{ element.Publisher }}, Link to License: {{element.linkToLicense}}, Resource: {{ element.resourceFor }}, Resource Type: {{ element.resourceType }}
       </div>
-      <div class="list-item-action">
-        <a mat-raised-button i18n [routerLink]="['/resources/view', item.doc._id]">View Details</a>
-        <a mat-raised-button i18n [routerLink]="['/resources/wip']">Add To Library</a>
-        <a mat-raised-button i18n [routerLink]="['/resources/wip']">Feedback</a>
-        <button mat-raised-button color="warn" (click)="deleteClick(item.doc)" i18n>Delete</button>
-      </div>
-    </div>
-    <div class="list-item-rating">
-      <span class="stars">
-        <!-- used simple arithmetic to get 20 and converted into % -->
-        <span [ngStyle]="{'width': 20 * getRating(item.doc.sum,item.doc.timesRated) + '%'}"></span>
-      </span>
-      <div class="label">
-        <span class="small">Rating ({{ rating }}) </span>
-        <span class="small">Female ({{ fRating }}%)
-        </span>
-        <span class="small">Male ({{ mRating }}%)
-        </span>
+      <div class="list-item-content">
+        <a mat-raised-button color="primary" i18n [routerLink]="['/resources/view', element._id]">View Details</a>
+        <a mat-raised-button color="primary" i18n [routerLink]="['/resources/wip']">Add To Library</a>
+        <a mat-raised-button color="primary" i18n [routerLink]="['/resources/wip']">Feedback</a>
+        <button mat-raised-button color="warn" (click)="deleteClick(element)" i18n>Delete</button>
       </div>
-      <div class="info">
-        <span> {{ rating }}
+    </mat-cell>
+  </ng-container>
+  <ng-container matColumnDef="rating">
+    <mat-header-cell *matHeaderCellDef></mat-header-cell>
+    <mat-cell *matCellDef="let element">
+      <div class="list-item-rating">
+        <span class="stars">
+          <!-- used simple arithmetic to get 20 and converted into % -->
+          <span [ngStyle]="{'width': 20 * getRating(element.sum, element.timesRated) + '%'}"></span>
         </span>
-        <div class="rating-bar">
-          <!-- used ngStyle here to set the width of the bar dynamically -->
-          <div [ngStyle]="{ 'background-color': 'gold', 'height': '8px', 'width': fRating + '%'}"></div>
+        <div class="label">
+          <span class="small">Rating ({{ rating }})</span>
+          <span class="small">Female ({{ fRating }}%)</span>
+          <span class="small">Male ({{ mRating }}%)</span>
         </div>
-        <div class="rating-bar">
-          <div [ngStyle]="{ 'background-color': 'gold', 'height': '8px', 'width': mRating + '%'}"></div>
+        <div class="info">
+          <span> {{ rating }}
```

Comment:
should be in a single line. Closing tag of span is in another line , that needs to be fix. Please check it once .